doc/source/whatsnew/v1.3.0.rst

```diff
@@ -647,6 +647,7 @@ Deprecations
 - Deprecated setting :attr:`Categorical._codes`, create a new :class:`Categorical` with the desired codes instead (:issue:`40606`)
 - Deprecated behavior of :meth:`DatetimeIndex.union` with mixed timezones; in a future version both will be cast to UTC instead of object dtype (:issue:`39328`)
 - Deprecated using ``usecols`` with out of bounds indices for ``read_csv`` with ``engine="c"`` (:issue:`25623`)
+- Deprecated passing arguments as positional in :meth:`DataFrame.drop_duplicates` and :meth:`Series.drop_duplicates` (:issue:`41485`)
```

Comment:
perhaps mention that `subset` is allowed (e.g. "except for ``subset``"), other than that, if the tests all pass, this looks good to me

Reply:
Alright, thanks, I made that change and committed again.

I have checked that all tests on the methods (not just mine) still pass without issue also.